Sources/Conformance/conformance.pb.swift

```diff
@@ -175,15 +175,15 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message, SwiftProtobuf.Prot
       return nil
     }
 
-    fileprivate func traverse(visitor: SwiftProtobuf.Visitor, start: Int, end: Int) throws {
+    fileprivate func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V, start: Int, end: Int) throws {
```

Comment:
The PR as a whole cuts binary size by about 4k for 100 fields (regardless of field type).  I didn't measure the impact for each separate change in this case, but I believe when I did a similar rearchitecture on the decoder side, that the generic here had no impact.